cra/sample-cra-ci/ci-pipeline.yaml

```diff
@@ -0,0 +1,156 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: ci-pipeline
+spec:
+  params:
+    - name: repository
+      description: the git repo
+    - name: branch
+      description: branch
+    - name: revision
+      description: the git revision/commit for the git repo
+      default: ""
+    - name: repository-integration
```

Comment:
$ tekton-lint "cra/**/*.yaml" git/*.yaml toolchain/*.yaml
cra/sample-cra-ci/ci-pipeline.yaml:
warning (15,7,17,1): Pipeline 'ci-pipeline' defines parameter 'repository-integration', but it's not used anywhere in the spec